### PR TITLE
CVSB-11826 - correct visit time displayed in atf email

### PR DESCRIPTION
**Pull request by castancu**
Ticket that ensures the correct display of times in the ATF email.

https://jira.dvsacloud.uk/browse/CVSB-11826

### CODE REVIEW

File: src/utils/generateNotificationData.ts [hunk 1]
```diff
@@ -24,7 +24,7 @@ class NotificationData {
         const axlesSeats = (event.activity.vehicleType === VEHICLE_TYPES.PSV) ? event.activity.numberOfSeats : event.activity.noOfAxles;
         const vrmTrailerId = (event.activity.vehicleType === VEHICLE_TYPES.TRL) ? event.activity.trailerId : event.activity.vrm;
         personalization.activityDetails += `^#${this.capitalise(event.activityType)} (${vrmTrailerId})
-      ^• Time: ${this.formatDateAndTime(event.activity.testTypes.testTypeStartTimestamp, "time")} - ${this.formatDateAndTime(event.activity.testTypes.testTypeEndTimestamp, "time")}
+      ^• Time: ${this.formatDateAndTime(event.activity.testStartTimestamp, "time")} - ${this.formatDateAndTime(event.activity.testEndTimestamp, "time")}
```
Comment (JonLazariniDD) [inline src/utils/generateNotificationData.ts:27]: I think I will need to have a catch up to understand the event better. My understanding is that  the even returns a list of activities and I can see that we have the following keys (from activity services depending on the type of visit) `startTime` and `endTime`.

Comment (bristol-mike-jones) [reply src/utils/generateNotificationData.ts:27]: the activity object here is either an activity or a test result object, with the `IActivitiesList.activityType` indicating which.  So at this point, we are just assuming that the activity object is a test result, and has these attributes.  Which tbf it should.

Using` testStartTimestamp`/ and `testEndTimestamp` aligns to the retro key report, so it explains why this fixes the issue.

**Review by bristol-mike-jones — APPROVED** [review]

File: src/utils/generateNotificationData.ts (hunk 1 above)
Comment (JonLazariniDD) [reply src/utils/generateNotificationData.ts:27]: Thank you 👍

**Review by JonLazariniDD — APPROVED** [review]

**Review by bristol-mike-jones — APPROVED** [review]

File: src/utils/generateNotificationData.ts (hunk 1 above)
Comment (JonLazariniDD) [reply src/utils/generateNotificationData.ts:27]: one for the future, we could use object destructuring to make it more concise like that
```
const {activity: {testStartTimestamp, testEndTimestamp}} = event

// ...
formatDateAndTime(testStartTimestamp)
```

so we make it more concise and easier to read in the function call.

**Review by JonLazariniDD — APPROVED** [review]

**Pull request merged**